Readme content updates for 2.10 (#8451)
easydigitaldownloads/easy-digital-downloads: readme.txt

Here are some of the features supported out-of-the-box:

- **Frontend shopping cart** - Customers can browse your product catalogue, add items to their cart, and checkout when they are ready. Or they can bypass the cart using Buy Now buttons.
- **Highly customizable and clean checkout** - Buyers can complete their purchase with ease, directly on your site's checkout page.

> - **Payment flexibility** - Accept credit card payments using Stripe(https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included.

Reviewer: ```suggestion
- **Payment flexibility** - Accept credit card payments using [Stripe](https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included.
```

Reviewer 2: With the ordering of the language here, it's not clear that Apple/Google Pay are included via Stripe.

Reviewer 3: `Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons`

or something similiar

Author: @cklosowski Agreed with your edit.

Reviewer 3: Ok @pderksen I'll get it all committed.

Here are some of the features supported out-of-the-box:

- **Frontend shopping cart** - Customers can browse your product catalogue, add items to their cart, and checkout when they are ready. Or they can bypass the cart using Buy Now buttons.
- **Highly customizable and clean checkout** - Buyers can complete their purchase with ease, directly on your site's checkout page.

> - **Payment flexibility** - Accept credit card payments using Stripe(https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons.

Reviewer 2: ```suggestion
- **Payment flexibility** - Accept credit card payments using [Stripe](https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons.
```

* BitPay
* Coinbase

> > What can I say? Stripe is my go-to payment processor and without this integration I don't know what I would do. Connect EDD to your Stripe account in seconds and you're done! Having a simple test mode toggle is also super valuable.

Reviewer: This is a direct quote, right? Should we be using square brackets to denote our changes?

Reviewer 3: If this is a direct quote from someone, we shouldn't modify it other than contextual hints we add, which should be in brackets.

Author: Assuming it's a direct quote (but I'm not sure), sure let's put "integration" in square brakets.

The second sentence can be changed back to the original.

> Having a simple test mode toggle is also a valuable development tool.

And simply leave off the 3rd sentence.

Reviewer 3: Ok, these are direct quotes from customers. I'll remove that 3rd sentence and put `integration` in square brackets.

With add-on plugins from our [extensions catalogue](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) you can super-charge your digital store. Some of our extension highlights include:

> * [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% fee we charge as well as adds the ability to accept pre-approved payments (extension must be installed and activated to remove the additional 2% fee)

Reviewer 2: @pderksen does this need to say "we charge" as it's not really us charging; it's an additional fee collected by Stripe? Also the parenthetical ends up feeling a bit redundant. Maybe simplify to this instead?

>Stripe Pro removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)

Author: @robincornett Yes much better wording. Approved!

> * [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% fee we charge as well as adds the ability to accept pre-approved payments (extension must be installed and activated to remove the additional 2% fee)

Reviewer 2: ```suggestion
* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)

```

With add-on plugins from our [extensions catalogue](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) you can super-charge your digital store. Some of our extension highlights include:

> * [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)

Reviewer 2: I think we want to double check the UTM tags in here please. Is that a @pderksen question?

Author: @robincornett Good catch. If you could please update to `&utm_content=stripe-pro`

Reviewer 2: ```suggestion
* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=stripe-pro&utm_campaign=readme) removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)
```
